scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -1,18 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-SPDX-FileCopyrightText: 2021 Alliander N.V.
-
-SPDX-License-Identifier: Apache-2.0
--->
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="https://www.lfenergy.org/compas/extension/v1"
-           targetNamespace="https://www.lfenergy.org/compas/extension/v1" elementFormDefault="qualified"
-           attributeFormDefault="unqualified" version="v1">
+<xs:schema xmlns:scl="http://www.iec.ch/61850/2003/SCL" xmlns="https://www.lfenergy.org/compas/extension/v1" xmlns:compas="https://www.lfenergy.org/compas/extension/v1" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" targetNamespace="https://www.lfenergy.org/compas/extension/v1" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1">
+    <xs:import namespace="http://www.iec.ch/61850/2003/SCL" schemaLocation="SCL.xsd"/>
     <xs:annotation>
         <xs:documentation xml:lang="en">
-            COPYRIGHT (c) 2021 Alliander N.V.
-        </xs:documentation>
-        <xs:documentation xml:lang="en">
-            Version 1.0 Release 2021/06/16
+            Version 2.0 Release 2022/01/17
```

Comment:
Change it to version 1.1, because it isn't a breaking change, just adding definitions.